pkg/asset/installconfig/azure/session.go

```diff
@@ -14,74 +14,98 @@ import (
 	"gopkg.in/AlecAivazis/survey.v1"
 )
 
-var authFileLocation = os.Getenv("HOME") + "/.azure/osServicePrincipal.json"
+const azureAuthEnv = "AZURE_AUTH_LOCATION"
+
+var authFilePath = os.Getenv("HOME") + "/.azure/osServicePrincipal.json"
 
 //Session is an object representing session for subscription
 type Session struct {
-	SubscriptionID string
-	Authorizer     autorest.Authorizer
+	Authorizer  autorest.Authorizer
+	Credentials Credentials
+}
+
+//Credentials is the data type for credentials as undestood by the azure sdk
+type Credentials struct {
+	SubscriptionID string `json:"subscriptionId,omitempty"`
+	ClientID       string `json:"clientId,omitempty"`
+	ClientSecret   string `json:"clientSecret,omitempty"`
+	TenantID       string `json:"tenantId,omitempty"`
 }
 
-// GetSession returns an azure session by checking credentials
+// GetSession returns an azure session by using credentials found in ~/.azure/osServicePrincipal.json
 // and, if no creds are found, asks for them and stores them on disk in a config file
 func GetSession() (*Session, error) {
-	err := getCredentials()
-	if err != nil {
-		return nil, err
-	}
+	//sets up openshift credential file for autorest
```

Comment:
hmm maybe i have it wrong, but initial testing

```go
package main

import "os"

func main() {
	os.Setenv("TEST_ENV", "set-by-go-binry")
}
```
```console
$ go build -o env env.go
$ ./env
$ echo $TEST_ENV
```
so you cannot temper with user's env

Reply:
ah, so I can remove this. thanks.

Reply:
done